### PR TITLE
DOC: interpolate: add an example of splPrep/PPoly.from_spline interop

### LINKED ISSUE #14348: Extract spline coefficient from splprep: tck
I am using python3, Scipy

I have a 3d points (x,y,z] From them I make s a spline using scipy.interpolate.splprep dunction

```
x_points = np.linspace(0, 2*np.pi, 10)
y_points = np.sin(x_points)
z_points = np.cos(x_points)
path = np.vstack([x_points, y_points, z_points])
tck, u = sc.splprep(path, k=3, s=0)
```

I wish to get the coefficients of the spline[i].
For example, the first spline:
sp_y0 = ay0 + by0(x-x4) + cy0(x-x4)^2 + dy0(x-x4)^3
sp_z0 = az0 + bz0(x-x4) + cz0(x-x4)^2 + dz0(x-x4)^3
to the second spline: [ay1, by1, cy1, dy1], [az1, bz1, cz1, dz1]

I used PPoly.from_spline
```
t, c, k = tck
cc = np.asarray(c)    # cc.shape is (3, 10) now
spl0 = sc.PPoly.from_spline((t, cc.T[0], k))
```
the coefficient suppose to be in `spl0.c`
but I'm expecting to get 8 coefficients for each spline, 4 for the sp_y, and 4 for sp_z

And please can you explain the parameter tck: t?
why are the first falues of the vector of knots are the same.
for the example above: 

> t=
> array([0.        , 0.        , 0.        , 0.        , 0.21757366,
>        0.34535401, 0.45402382, 0.54597618, 0.65464599, 0.78242634,
>        1.        , 1.        , 1.        , 1.        ])

Thanks

### PR DESCRIPTION
**Pull request by ev-br**
#### Reference issue

closes gh-14348

#### What does this implement/fix?

Give  `PPoly.from_spline` a docstring example of using it for parametric splines. There are two issues: `from_spline` only accepts 1D arrays of coefficients, hence in the N-D case users need to loop over the dimensions. Second is that `splPrep` returns the coefficients in a length-N list of arrays and this is incompatible with BSpline and PPoly which use *trailing* dimensions. This is a known wart, and it cannot be fixed in a reasonable way.

An alternative of vectorizing from_spline is not really viable because it needs to accept both tck-tuples (list-of-arrays) and BSpines (trailing dimensions). There is no reliable way to decide which convention to use. 

Thus, document the workaround and leave it at that.

[skip actions]

### CODE REVIEW

**Comment by mdhaber** [comment]
Re-running CI here since this was submitted when everything was red.

**Comment by ev-br** [comment]
Rebased on main locally and force-pushed. Let's see what Circle CI tells us.

File: scipy/interpolate/_interpolate.py [hunk 1]
```diff
@@ -1333,6 +1333,49 @@ def from_spline(cls, tck, extrapolate=None):
...
+        >>> (t, c, k), u = splprep([x, y], s=0)
+
+        Note that ``c`` is a list of two arrays of length 11.
+
+        >>> unew = np.arange(0, 1.01, 0.01)
+        >>> out = splev(unew, (t, c, k))
+
+        To convert this spline to the power basis, we convert each
+        component of the list of b-spline coefficients, ``c``, into the
+        corresponding cubic polynomial.
```
Comment (mdhaber) [inline scipy/interpolate/_interpolate.py:1359]:
```suggestion
        corresponding cubic polynomial.
```
It tells us there is trailing whitespace : (

File: scipy/interpolate/_interpolate.py [hunk 2]
```diff
@@ -1333,6 +1333,49 @@ def from_spline(cls, tck, extrapolate=None):
             If bool, determines whether to extrapolate to out-of-bounds points
             based on first and last intervals, or to return NaNs.
             If 'periodic', periodic extrapolation is used. Default is True.
+
+        Examples
+        --------
+
+        This funcion only supports 1D splines out of the box.
```
Comment (mdhaber) [inline scipy/interpolate/_interpolate.py:1340]:
```suggestion
        This function only supports 1D splines out of the box.
```

Comment (mdhaber) [reply scipy/interpolate/_interpolate.py:1340]: I'd suggest adding a very elementary example of that and emphasizing that the next example is for a more complex case. 

```
# <elementary example here>

This function only supports 1D splines out of the box, but ...
```

Comment (ev-br) [reply scipy/interpolate/_interpolate.py]: Done in fa842bb

**Review by rgommers — APPROVED** [review]
Looks great, thanks @ev-br!

**Pull request merged**